db-populate.sh

```diff
@@ -129,14 +129,16 @@ echo "\$environment: '$environment' \$password_for_users: '$password_for_users'
 HOST=mongo1
 NETWORK=opencrvs_default
 
-docker run --rm --network=$NETWORK mongo:4.4 mongo hearth-dev --host $HOST --eval "db.dropDatabase()"
-docker run --rm --network=$NETWORK mongo:4.4 mongo user-mgnt --host $HOST --eval "db.dropDatabase()"
-docker run --rm --network=$NETWORK mongo:4.4 mongo application-config --host $HOST --eval "db.dropDatabase()"
-docker run --rm --network=$NETWORK mongo:4.4 mongo metrics --host $HOST --eval "db.dropDatabase()"
-docker run --rm --network=$NETWORK mongo:4.4 mongo reports --host $HOST --eval "db.dropDatabase()"
-docker run --rm --network=$NETWORK mongo:4.4 mongo config --host $HOST --eval "db.dropDatabase()"
-docker run --rm --network=$NETWORK mongo:4.4 mongo webhooks --host $HOST --eval "db.dropDatabase()"
-docker run --rm --network=$NETWORK mongo:4.4 mongo performance --host $HOST --eval "db.dropDatabase()"
+docker run --rm --network=$NETWORK mongo:4.4 mongo --host $HOST --eval "\
+db.getSiblingDB('hearth-dev').dropDatabase();\
+db.getSiblingDB('openhim-dev').dropDatabase();\
+db.getSiblingDB('user-mgnt').dropDatabase();\
+db.getSiblingDB('application-config').dropDatabase();\
+db.getSiblingDB('metrics').dropDatabase();\
+db.getSiblingDB('config').dropDatabase();\
+db.getSiblingDB('performance').dropDatabase();\
+db.getSiblingDB('webhooks').dropDatabase();"
+
```

Comment:
Currently we don't clear `openhim-dev` in this script. But it was added in this change. .Not sure why it was not here before?